docs/configuration.v2alpha1.md

```diff
@@ -179,6 +179,7 @@ spec:
 | global.endpoint.credentials.appSecret.keyName | KeyName is the key of the secret to use. |
 | global.endpoint.credentials.appSecret.secretName | SecretName is the name of the secret. |
 | global.endpoint.url | URL defines the endpoint URL. |
+| global.env |  |
```

Comment:
Refer to previous comment to automatically fill out that field with `make generate`